using-mobile-extensions/adobe-media-analytics/README.md

```diff
@@ -0,0 +1,155 @@
+
+#### **Player Name**
+
+Name of the media player in use (e.g., "AVPlayer", "Native Player", "Custom Player").
+
+#### **Application Version**
+
+The version of the media player application/SDK.
+
+#### **Debug Logging**
+
+Enables or disables Media SDK logs.
+
+{% hint style="danger" %}
+This should be disabled for your production application.
+{% endhint %}
+
+### Add Media Analytics to your app
+
+{% hint style="info" %}
+This extension requires the [Adobe Analytics Extension](../adobe-analytics/README.md). You must add the Analytics extension to your Launch property, and make sure the extension is configured correctly.
+{% endhint %}
+
+{% tabs %}
+{% tab title="Android" %}
+Add the Media extension to your project using the app's Gradle file.
+
+#### Java
+
+1. Import the Media extension in your application's main activity.
+
+```java
+import com.adobe.marketing.mobile.*;
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+Add the library to your project via your Cocoapods `Podfile` by adding `pod 'ACPMedia'`.
+
+#### Objective-C
+
+```objectivec
+#import <ACPMedia.h>
+```
+
+#### Swift
+
+```swift
+import ACPMedia
+```
+
+{% endtab %}
+{% endtabs %}
+
+### Register Media with Mobile Core
+
+{% tabs %}
+{% tab title="Android" %}
+
+#### Java
+
+To register media with Mobile Core, call the `setApplication()` method in `onCreate()`, then call setup methods, as shown in this sample:
+
+```java
+public class MobileApp extends Application {
+
+  @Override
+  public void onCreate() {
+      super.onCreate();
+      MobileCore.setApplication(this);
+
+      try {
+          Media.registerExtension();  //Register Media with Mobile Core
+          Analytics.registerExtension();
+          Identity.registerExtension();
+      } catch (Exception e) {
+
+      }
+  }
+}
+```
+
+{% endtab %}
+
+{% tab title="iOS" %}
+In your app's `application:didFinishLaunchingWithOptions`, register Analytics with Mobile Core:
+
+#### Objective-C
+
+```objectivec
+- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+    [ACPMedia registerExtension];  //Register Media with Mobile Core
+    [ACPAnalytics registerExtension];
+    [ACPIdentity registerExtension];
+    return YES;
+  }
+```
+
+#### Swift
+
+```swift
+func application(_ application: UIApplication, 
+                 didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    ACPMedia.registerExtension();  //Register Media with Mobile Core
+    ACPAnalytics.registerExtension();
+    ACPIdentity.registerExtension();
+    return true;
+}
+```
+
+{% endtab %}
+{% endtabs %}
+
+## Configuration Keys
+
+If you need to update your SDK configuration programmatically, please use the following information to change your Media configuration values. For more information, see [Configuration Methods Reference](../mobile-core/configuration-reference/#update-configuration).
```

Comment:
- Please delete "If you need" from the beginning of the 1st sentence so that it reads "To update your SDK configuration..."

- Please remove "please".